openedx/core/djangoapps/courseware_api/tests/test_views.py

```diff
@@ -60,17 +64,17 @@ class CourseApiTestViews(BaseCoursewareTests):
     Tests for the courseware REST API
     """
     @ddt.data(
-        (True, None, False),
-        (True, 'audit', False),
-        (True, 'verified', False),
-        (False, None, False),
-        (False, None, True),
+        (True, None, ACCESS_DENIED),
```

Comment:
check_public_access now returns AccessResponse objects instead of simple booleans.  These evaluate as booleans (they have a `__bool__`)